providers.tf

```diff
@@ -0,0 +1,28 @@
+terraform {
+
+  required_version = "= 1.5.7"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 5.17.0"
+    }
+    random = {
```

Comment:
do you really need this module?

Reply:
Instead of using the random module, we could set values manually through values to pass predefined parameters directly. This would simplify the configuration by allowing controlled values, and we could remove the random module.